HW0/P5/P5.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+    x = np.arange(1,500)
+    y1 = x - 1
+    y2 = np.ceil(np.log2(x))
```

Comment:
The np.ceil function is used here because you can't evenly divide 256 bags across 3 cashiers (it would be most efficient to use numbers of cashiers that are exponents of 2).